src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -990,6 +990,23 @@ public void reset() {
         public void showNotificationIfAvailable(Activity parent);
 
         /**
+         * Applies A/B test changes, if they are present. By default, your application will attempt
+         * to join available experiments any time an activity is resumed, but you can disable this
```

Comment:
I know I can test it by myself. But, if I have multiple experiments under the same project, will all the changes be loaded to the app? I guess so though..